man/org.freedesktop.systemd1.xml

```diff
@@ -3471,6 +3473,8 @@ node /org/freedesktop/systemd1/unit/avahi_2ddaemon_2eservice {
 
     <!--property OpenFile is not documented!-->
 
+    <!--property ExtraFileDescriptorNames is not documented!-->
```

Comment:
can you add a brief sentence what this is? and highlight the fact we don't expose the fd when reading the prop, and that things are a bit asymmetric here.